Makefile.helpers

```diff
@@ -6,7 +6,7 @@ EDITOR ?= vim
 
 SHELL = /bin/bash
 
-DEFAULT_HELP_TARGET ?= help
```

Comment:
If we deprecate this, we need to update dependencies: 

https://github.com/search?q=org%3Acloudposse+DEFAULT_HELP_TARGET&type=Code

Reply:
That feature reverted